host-configs/lassen-rhel-gcc-gpu.cmake

```diff
@@ -0,0 +1,26 @@
+# c compiler
+#set(CMAKE_C_COMPILER "/usr/tce/packages/clang/clang-coral-2018.08.08/bin/clang" CACHE PATH "")
+set(CMAKE_C_COMPILER "gcc" CACHE PATH "")
+
+# cpp compiler
+#set(CMAKE_CXX_COMPILER "/usr/tce/packages/clang/clang-coral-2018.08.08/bin/clang++" CACHE PATH "")
```

Comment:
Should we keep the clang compilers in this host config?

Reply:
Clang is no longer required. Cleaned up the hostfile.